README.md

```diff
@@ -157,27 +114,19 @@ $ ansible-playbook -i localhost, --connection=local \
         kubevirt_mf=kubevirt.yaml" \
         install-kubevirt-on-openshift.yml
 ```
+where
+* `kubevirt_mf` is a path to KubeVirt manifest. See [releases](https://github.com/kubevirt/kubevirt/releases) or build it from sources,
+* `openshift_ansible_dir` is a path to a cloned [OpenShift Ansible][openshift-ansible-project] git repository,
+* `kconfig` is a path to a kube configuration file.
 
 [![asciicast](https://asciinema.org/a/161278.png)](https://asciinema.org/a/161278)
 
 
-## Deploy new Kubernetes or OpenShift cluster and KubeVirt with Lago
-
-This section describes how to deploy KubeVirt or OpenShift cluster with KubeVirt installed on it using Lago.
-
-### Parameters
+## Deploy a new Kubernetes or OpenShift cluster and KubeVirt with Lago
 
-| Name             |  Value        | Description                            |
```

Comment:
I rather prefer table for parameters, but if ppl find that list form more readable let it be then.